primes.py

```diff
@@ -1,16 +1,24 @@
 """
-Your module documentation here
+Modulo de ejercicio para examen
 """
 
 
 class PrimeClass(object):
     """
-    Your class documentation here
+    Esta clase es creada para verificar numeros primos
     """
 
     def is_prime(self, num_int):
         """
-        Your method documentation here
+        Este metodo devuelve un True en caso de que el numero
+        proporcionado sea primo
         """
-        # your primes code here
-        return "not implement yet"  # Remove this dummy line
+        if num < 1:
```

Comment:
This variable must be the num_int isn't ?, the variable num don't exist in this method, please check it because you have a syntax error here.